src/test/regress/README.md

```diff
@@ -81,6 +81,35 @@ Adding a new test file is quite simple:
 4. Check that the output is as expected
 5. Copy the `.out` file from `results` to `expected`
 
+### Adding an alternative test output
+
+Sometimes the test output changes because we run tests in different configurations.
+The most common example is an output that changes in different Postgres versions.
+If we don't care about the change, we try to add [a normalization rule](https://github.com/citusdata/citus/blob/main/ci/README.md#normalize_expectedsh), or find
+ways to avoid the output altogether.
+
+If we actually care about the output in all its forms, we add an alternative test
+output. In order to maintain a clean test suite, make sure to explain in the test
+header the reason why it has an alternative output, and when we can drop the
+alternative output in the future.
+
+For example, the header of [multi_insert_select.sql](https://github.com/citusdata/citus/blob/main/src/test/regress/sql/multi_insert_select.sql) is as follows:
```

Comment:
But we'll drop the alternative output for `multi_insert_select.sql` when we drop PG14 support, right? We'll need to update this link then. Maybe just say `For example:` without a link?